csvs/nexo-transaction.json

```
@@ -0,0 +1,472 @@
+{
+  "id": "next-transaction",
+  "service": "nexo",
+  "formatter": {
+    "Transaction": ["string"],
+    "Type": ["string"],
+    "Input Currency": ["string"],
+    "Input Amount": ["big-decimal"],
+    "Output Currency": ["string"],
+    "Output Amount": ["big-decimal"],
+    "Fee": ["string", "big-decimal"],
+    "Fee Currency": ["string"],
+    "Details": ["string"],
+    "Date / Time (UTC)": ["iso8601"]
+  },
+  "parser": {
+    "parsers": [
+      {
+        "description": "Interest / Fixed Term Interest",
+        "conditions": [
+          {
+            "value": {
+              "type": "cell",
+              "column": "Type"
+            },
+            "pattern": {
+              "type": "in",
+              "values": ["Interest", "Fixed Term Interest"]
+            }
+          },
+          {
+            "value": {
+              "type": "cell",
+              "column": "Output Currency"
+            },
+            "pattern": {
+              "type": "defined"
+            }
+          },
+          {
+            "value": {
+              "type": "cell",
+              "column": "Output Amount"
+            },
+            "pattern": {
+              "type": "defined"
+            }
+          },
+          {
+            "value": {
+              "type": "cell",
+              "column": "Date / Time (UTC)"
+            },
+            "pattern": {
+              "type": "defined"
+            }
+          }
+        ],
+        "generators": [
+          {
+            "type": "staking-reward",
+            "source": {
+              "type": "join",
+              "values": [
+                { "type": "service" },
+                "_",
+                { "type": "cell", "column": "Date / Time (UTC)" }
```

```suggestion
                { "type": "cell", "column": "Transaction" }
```

Transaction が unique なのであれば、 unique の値の方がより良いかも

---

https://github.com/0xtorch/datasource/pull/9189/commits/eb9e83e12331501291e926f76c0c6e4eac310218 で対応